life_expectancy/cleaning.py

```diff
@@ -1,133 +1,110 @@
 """Module to clean assignment data"""
+
 import os
 import re
 import argparse
 import pandas as pd
 from pandas import DataFrame
 
 
-def load_raw_data(
-    raw_data_file_path: str
-):
+def load_data():
     """
     This function is responsible for loading the raw data,
     applying the correct split and tranforming it to a pandas dataframe
 
-    param: raw_data_file_path -> path to the raw data file
-
-    return: Pandas dataframe with columns properly defined
+    return: eu_life_expectancy_raw_dataframe -> pandas dataframe
     """
-    os_path_pardir = os.path.dirname(__file__)
+
+    #Get parent directory
+    parent_directory = os.path.dirname(__file__)
+    raw_data_file_path = '/data/eu_life_expectancy_raw.tsv'
 
     #Read tsv file
-    splitted_file = []
-    with open(os_path_pardir + raw_data_file_path, encoding="utf-8") as file:
-        delimiters = ',', '\\time\t', '\t', '\n'
-        regex_pattern = '|'.join(map(re.escape, delimiters))
-        #Read first line
-        for line in file:
-            splitted_line = re.split(regex_pattern,line)
-            splitted_file.append(splitted_line[:-1])
+    file_with_split_lines = []
+    with open(parent_directory + raw_data_file_path,
+              encoding="utf-8") as file:
 
-    eu_life_expectancy_raw_data = pd.DataFrame(splitted_file[1:], columns = splitted_file[0])
+        #Get list of file delimiters
+        delimiters = [',', '\\time\t', '\t', '\n']
+        delimiters_pattern = '|'.join(map(re.escape, delimiters))
 
+        #Split file line by line
+        for line in file:
+            splitted_line = re.split(delimiters_pattern, line)
+            file_with_split_lines.append(splitted_line[:-1]) #Exclude \n last split
 
-    return eu_life_expectancy_raw_data
+    #Create pandas dataframe
+    eu_life_expectancy_raw_dataframe = pd.DataFrame(data = file_with_split_lines[1:],
+                                               columns = file_with_split_lines[0])
```

Comment:
To avoid all the previous code treating the file line by line, you can use:
```suggestion
    eu_life_expectancy_raw_dataframe = pd.DataFrame(os_path_par_dir + raw_data_file_path, sep=',| \t|\t', engine='python')
```

Reply:
If you accept the suggestion above, you can delete all previous columns. Otherwise, I would recommend encapsulating them as part of a different function